test-em-all.bash

```
@@ -0,0 +1,242 @@
+  recreateComposite "$PROD_ID_REVS_RECS" "$body"
+
+}
+
+set -e
+
+echo "Start Tests:" `date`
+
+echo "HOST=${HOST}"
+echo "PORT=${PORT}"
+
+if [[ $@ == *"start"* ]]
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)